tests/time_tests/test_runner/conftest.py

```diff
@@ -273,8 +251,8 @@ def prepare_db_info(request, test_info, executable, niter, manifest_metadata):
             test_info["db_info"].update(json.load(db_meta_f))
 
     # add cpu cache status
-    cpu_cache = True if request.config.getoption("cpu_cache") else False
-    test_info["db_info"].update({"use_cpu_cache": cpu_cache})
+    cpu_cache = True if request.config.getoption("model_cache") else False
```

Comment:
cpu_cache -> model_cache

Reply:
No need in if/else, `request.config.getoption("model_cache")` should return True/False

Reply:
done